add css and scripts config example (#22)
ConsenSys/doctools.template-site: docs/create/advanced/add-css-and-js.md

---
description: How to add CSS and Javascript
---

# Add custom CSS and Javascript

> The following guidelines explain how to add CSS and Javascript to your site without modifying

Reviewer: ```suggestion
The following guidelines explain how to add CSS and JavaScript to your site without modifying
```

> # Add custom CSS and Javascript
> 
> The following guidelines explain how to add CSS and Javascript to your site without modifying
> the common doc theme.
> 
> Extra script and style configuration enables individual sites to have a specific colour scheme
> or run some script to complement the initial set provided by the common theme.
> 
> These scripts and styles will be loaded on all pages of the doc site.

Reviewer 2: ```suggestion
# Add custom CSS and JavaScript

The following guidelines explain how to add CSS and JavaScript (JS) to documentation sites that use the [new system](../../overview/index.md#new-documentation-system) without modifying the common documentation theme.

Extra script and style configuration enables individual sites to have a specific color scheme
or to run scripts in addition to the common set of scripts.

These scripts and styles are loaded on all pages of the documentation site.
```

Add scripts and styles by updating the `mkdocs.extra.yml` configuration file.

!!! example "Custom CSS and JS files example"

    Create new `docs/assets/stylesheets/` and `docs/assets/javascripts/` folders.

>     Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a javascript file  `docs/assets/javascripts/a_custom_script.js`.

Reviewer: ```suggestion
    Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a JavaScript file  `docs/assets/javascripts/a_custom_script.js`.
```

>     Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a javascript file  `docs/assets/javascripts/a_custom_script.js`.
> 
>     Modify the `mkdocs.extra.yml` with the following:

Reviewer 2: ```suggestion
    Create a CSS file `docs/assets/stylesheets/a_custom_style.css` and a JS file  `docs/assets/javascripts/a_custom_script.js`.

    Modify `mkdocs.extra.yml` with the following:
```
